providers/sh_cluster.rb

```diff
@@ -0,0 +1,44 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# Provider:: sh_cluster
+#
+
+use_inline_resources
+
+action :initialize do
+  search_heads = new_resource.search_heads
+  admin_password = new_resource.admin_password
+
+  execute 'Captain assignment' do
```

Comment:
Initially I had it. But then noticed that we are not doing it while pushing the [apply-cluster-bundle] (https://github.com/cerner/cerner_splunk/blob/stable/recipes/cluster_master.rb).